hack/clean.sh

```diff
@@ -0,0 +1,47 @@
+#!/bin/bash
+
+# **clean.sh**
+
+cd $GOPATH/src/github.com/kubeedge/kubeedge
+
+removefolder() {
+if [ -d bin ]; then
+rm -rf bin
+fi
+
+if [ -d .idea ]; then
+rm -rf .idea
+fi
+}
+
+#terminate mosquitto if running
+killmosquitto() {
+if  pgrep mosquitto  >/dev/null 2>&1  ; then
+     pkill -9 mosquitto
+fi
+
+}
+
+#terminate edge core if running
+killedge () {
+if pgrep edge_core >/dev/null 2>&1 ; then
+     pkill -9 edge_core
+fi
+
+}
+
+#delete all logs
```

Comment:
comment says only logs , but the function does a lot more. please check and improve the comment.